event/client.go

```diff
@@ -14,12 +14,12 @@ type Client struct {
 	Key string
 }
 
-// Get returns the details of an event
-// For more details see https://stripe.com/docs/api#retrieve_event.
+// Get returns the details of an event.
 func Get(id string, params *stripe.EventParams) (*stripe.Event, error) {
 	return getC().Get(id, params)
 }
 
+// Get returns the details of an event.
```

Comment:
do we care about this wording? We don't always say "the details of". I think it's mostly fine but wanted to flag

Reply:
Yeah, seems fine to me, especially in areas like this where the resource name is somewhat vague.